### DISCUSSION TITLE
Uniswap v4: A game-changing evolution from v3

### DISCUSSION THREAD

**Discussion by serrq** [horizontalsystems/unstoppable-wallet-android#6229] [General]
https://www.cryptonewsz.com/uniswap-v4-a-game-changing-evolution-from-v3/

**Comment by Istiagilter** [comment]
Hope hooks will be such great thing as described everywhere.

**Reply by anthony774882** [reply]
cool idea!

**Reply by 2Mouroso** [reply]
yeah thats right

**Reply by alexskin99** [reply]
Yes, good!

**Reply by biba1207** [reply]
That will be great!

**Reply by Van5205** [reply]
Yes, very good

**Reply by Johnick5** [reply]
Yes, very good! Its great idea

**Reply by Johnick55** [reply]
Yes, good!

**Reply by VanMix5** [reply]
Yes, very good

**Reply by Scottie33Pippen** [reply]
I waiting it

**Reply by lolyjul** [reply]
thank you! nice job

**Reply by anutkanusa** [reply]
Yep, great idea!

**Reply by Ellu39940** [reply]
really!

**Reply by asiahluciy** [reply]
Thanks!

**Reply by wenhaowenhao5** [reply]
thank youm, good job

**Reply by santoslopez17sl** [reply]
Thanks!

**Reply by keahinderi** [reply]
Thanks, Very good!

**Reply by jaggasarli** [reply]
Thanks!

**Reply by sakianando** [reply]
Thanks

**Reply by GGGrany** [reply]
Thanks

**Reply by svzaya** [reply]
Yes, very good

**Reply by XXXAshley** [reply]
a really good news!

**Reply by yourluckyday1** [reply]
++ i hope its working

**Comment by kanuer** [comment]
hope

**Comment by KataKult** [comment]
We are very waiting of V4 update! In love with Uni

**Reply by LunaLi777** [reply]
yes, that's right

**Reply by Luchia777** [reply]
Absolutely

**Reply by ranyaali42** [reply]
Yes

**Comment by Abmarss** [comment]
I am so glad about this release! Very excited with Uniswap V4

**Reply by jameyalaid** [reply]
Me too!

**Reply by nvernyacia** [reply]
Me too)

**Comment by macfarlanechristal** [comment]
Very nice

**Reply by prsm171** [reply]
Yes

**Reply by nomisweet232** [reply]
Very nice!

**Comment by DanaMont** [comment]
I am really very excited about V4. Waiting long time

**Comment by BellionSa** [comment]
V4 nice, very excited :3

**Reply by LunaLi777** [reply]
Me too.

**Comment by pdv196872** [comment]
Freedom drives Progress

**Comment by Vladislav3107** [comment]
hello

**Reply by wenaddiell** [reply]
Hello)

**Reply by blayblayday** [reply]
Hi

**Comment by gena983331** [comment]
It is interesting

**Reply by beni4large** [reply]
Very!

**Comment by sharronlonrusoilaf** [comment]
v3 is better

**Reply by vadimsenin26** [reply]
yep

**Reply by LunaLi777** [reply]
Really?

**Comment by tilkerinno** [comment]
niceeee

**Comment by skybermuber** [comment]
thx! rly!)

**Comment by cruzymarlanupton** [comment]
interesting, i need to check it

**Comment by MingmingShela** [comment]
expect

**Comment by spicedoubles** [comment]
It is interested

**Comment by emrekusak** [comment]
Uniswap çok başarılı bir proje ve platform. Ben iyi işler yapıcağına inanıyorum. Başarılar diliyorum.

**Comment by emrekusak** [comment]
Looking good news. I like it

**Comment by sadlymad** [comment]
nice nice

**Comment by masha22222** [comment]
Good!

**Comment by atlanticgg** [comment]
nice

**Comment by rostisi** [comment]
That's a game changer!

**Comment by MGA-debug** [comment]
thx!

**Comment by Amuba228** [comment]
We are very waiting of V4 update! In love with Uni

**Comment by ArutStory** [comment]
It looks very interesting

**Comment by ALLEx92** [comment]
Cool! Very
 Interesting

**Comment by Kaydenbre** [comment]
V4 it's a perfect protocol

**Comment by Tutassk** [comment]
Only like

**Comment by alenagreat** [comment]
Impressive!

**Comment by toderiko** [comment]
That is application or web?

**Comment by Olybok35** [comment]
Interesting information!

**Comment by codyhooover** [comment]
not bad. what the next?

**Comment by SerhiiBitcoinchenko** [comment]
thats cool!

**Comment by Breadmaker1** [comment]
Great job

**Comment by Dyorich** [comment]
sounds good

**Comment by Luchia777** [comment]
Its very exciting!

**Comment by raveauigotzb** [comment]
Waiting for updates! Thanks a lot)

**Comment by pafkakummu** [comment]
I am glad about this release! cool

**Comment by MaryTop1** [comment]
Good job.

**Comment by comp57** [comment]
Its cool

**Comment by Leanora33** [comment]
Thank you for your encouragement! Have a great day at work. Remember, your hard work and dedication are sure to lead to success. Keep going strong!

**Comment by gorbanyok** [comment]
nice work!

**Reply by kolos0911** [reply]
like it too

**Comment by cosmicexplorer47** [comment]
Wow, this is an impressive update! The progress you've made from v3 to v4 is truly remarkable. The new features and improvements you've introduced have the potential to revolutionize the decentralized finance space. Keep up the great work!

**Comment by AndreyPetrov74** [comment]
News are great!

**Comment by Elenadee** [comment]
hope it will work. thx

**Comment by Kasian8702** [comment]
Nice article and cool news

**Comment by Belonata** [comment]
Thx! Good job)

**Reply by ghel2201** [reply]
Thx! Good job)

**Comment by Hrybinchak** [comment]
Very glad to hear about it. Greetings to all!

**Comment by JohnS455** [comment]
Good idea.

**Comment by Zahangir1234** [comment]
well done guys

**Comment by masha22222** [comment]
hi

**Comment by Olybok35** [comment]
This is a breakthrough! It will be interesting!

**Comment by vshmal** [comment]
nice!!!keep going

**Reply by denverryan55** [reply]
Nice!

**Reply by birdbirdbird55** [reply]
Nice)

**Comment by HaugaTree** [comment]
one more great challenge!)))

**Comment by Scottie33Pippen** [comment]
one more good news

**Comment by Dilimed** [comment]
Nice

**Comment by GeorgiyWin** [comment]
really good idea

**Comment by Yuldaza** [comment]
Hi. Good

**Comment by ahollaniab** [comment]
Good job!

**Comment by arturbebov45** [comment]
turned out very cool

**Comment by davidweb3a** [comment]
thx! rly!)

**Comment by alinabdusat** [comment]
Hi! Thats good, continue the next

**Comment by vikserov14** [comment]
thx, Fine

**Comment by Kirill1837** [comment]
thx .thats intresting

**Comment by Holly112** [comment]
nice, thx!

**Comment by tanka678** [comment]
super

**Comment by maria2346c** [comment]
stars

**Comment by InfluentialThee** [comment]
good

**Comment by asavajames** [comment]
changing evolution from

**Comment by anetkit** [comment]
cool

**Comment by Djagernaut11** [comment]
good job

**Comment by MikeSoul21** [comment]
Really nice

**Comment by kewundef** [comment]
great

**Comment by Istiagilter** [comment]
Any advance of this story?
